core/arch/arm/kernel/thread.c

```diff
@@ -942,13 +936,15 @@ void thread_init_threads(void)
 	}
 }
 
-void thread_clr_thread_core_local(void)
+void thread_init_thread_core_local(void)
 {
 	size_t n = 0;
+	struct thread_core_local *tcl = thread_core_local;
 
-	for (n = 0; n < CFG_TEE_CORE_NB_CORE; n++)
-		thread_core_local[n].curr_thread = -1;
-	thread_core_local[0].flags |= THREAD_CLF_TMP;
+	for (n = 0; n < CFG_TEE_CORE_NB_CORE; n++) {
+		tcl[n].curr_thread = -1;
+		tcl[n].flags |= THREAD_CLF_TMP;
```

Comment:
`tcl[n].flags` is supposed to be `0` before this so we could use a direct assignment instead, unless I'm missing something.

Reply:
Indeed, I'll use `tcl[n].flags = THREAD_CLF_TMP;` instead.